operations/app/src/modules/key_vault/main.tf

```diff
@@ -15,10 +15,14 @@ resource "azurerm_key_vault" "application" {
   purge_protection_enabled        = true
 
   network_acls {
-    bypass                     = "AzureServices"
-    default_action             = "Deny"
+    bypass         = "AzureServices"
+    default_action = "Deny"
+
+    ip_rules = sensitive(concat(
```

Comment:
IP addresses are considered sensitive in all whitelisted rules, so the IP address is not leaked in logs.